lib/cache/Apc.class.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * Created by PhpStorm.
```

Comment:
please remove this comment. you should probably get phpstorm to stop inserting them